### PR TITLE
Config + terraform to create Azure verifier

### PR DESCRIPTION
**Pull request by alicefuzier**

### CODE REVIEW

File: bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/builder/BagVerifierWorkerBuilder.scala [hunk 1]
```diff
@@ -41,26 +44,40 @@ object BagVerifierWorkerBuilder {
...
+        )
+      case "standalone" =>
+        buildStandaloneVerifierWorker(
+          primaryBucket,
+          metricsNamespace = metricsNamespace,
+          alpakkaSqsWorkerConfig = alpakkaSqsWorkerConfig,
+          ingestUpdater,
+          outgoingPublisher
+        )
+      case _ => ???
```
Comment (alexwlchan) [inline bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/builder/BagVerifierWorkerBuilder.scala:80]: Throw an exception, probably?

Comment (alicefuzier) [reply bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/builder/BagVerifierWorkerBuilder.scala:80]: 😂  🤦‍♀️

File: bag_verifier/src/main/resources/application.conf [hunk 2]
```diff
@@ -4,5 +4,5 @@ aws.outgoing.sns.topic.arn=${?outgoing_topic_arn}
 aws.metrics.namespace=${?metrics_namespace}
 bag-verifier.primary-storage-bucket=${?primary_storage_bucket_name}
 operation.name=${?operation_name}
-bag-verifier.mode.is_replica=${?is_replica}
+bag-verifier.mode=${?bag_verifier_mode}
```
Comment (alexwlchan) [inline bag_verifier/src/main/resources/application.conf:7]: Do you need the azure endpoint?

Comment (alicefuzier) [reply bag_verifier/src/main/resources/application.conf:7]: yes. the PR is not ready yet. I have to fix a couple of things

File: terraform/modules/stack/variables.tf [hunk 3]
```diff
@@ -69,6 +69,15 @@ variable "replica_glacier_bucket_name" {
...
 }
 
+variable "azure_container_name" {
+  type = string
+}
+
+
+variable "azure_endpoint_sm_parameter" {
+  type = string
+}
```
Comment (alexwlchan) [inline terraform/modules/stack/variables.tf:79]: You already have azure_ssm_parameter a few lines down.

**Pull request merged**